website/src/libs/api-v3/amazonPayments.js

```diff
@@ -0,0 +1,93 @@
+import amazonPayments from 'amazon-payments';
+import nconf from 'nconf';
+import common from '../../../../common';
+let t = common.i18n.t;
+const IS_PROD = nconf.get('NODE_ENV') === 'production';
+
+let api = {};
+
+function connect (amazonPayments) { // eslint-disable-line no-shadow
```

Comment:
this function shouldn't be called every time. Just when the server starts and then store the return value somewhere